Dockerfile

```
@@ -20,13 +15,24 @@
 RUN mkdir /venv
 WORKDIR /venv
-COPY pyproject.toml .
+COPY pyproject.toml poetry.lock ./
+
+# Install dependencies
 RUN poetry config virtualenvs.in-project true
+RUN poetry lock
 RUN poetry install --no-root
 
+# Set up application directory
 RUN mkdir /app
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)